examples/aggregates/README.md

```diff
@@ -1,60 +1,49 @@
 # Astarte Aggregates example
 
 This example will guide you through setting up a simple ESP32 application running the Astarte SDK
-and interacting with a remote instance of Astarte.
+and interacting with a local instance of Astarte.
 
-The example device to perform the following actions:
-- Connect to a remote Astarte instance using a manually generated Device ID and credential secret.
-- When new aggregated object values are published on a server owned interface, then the device will
-publish new aggregated object values on the device owned interface.
-
-If you want to start using a local instance of Astarte hosted on your machine and register the
-device automatically you can check out the [`toggle_led` example](../toggle_led) in this repository.
+The example device will perform the following actions:
+- Connect to a local Astarte instance using a manually generated Device ID and credential secret.
+- When new aggregated object values are published on a server-owned interface, then the device will
+publish new aggregated object values on the device-owned interface.
 
 We will assume you already performed the common configuration and satisfy the common prerequisites.
-
-## Device connection to Astarte
-
-Once the example has been flashed it will automatically attempt to connect the device to the remote
-instance of Astarte.
-Verify this by using the monitor functionality of the ESP-IDF to observe the log messages of the
-ESP32.
-After connection has been accomplished, the following ESP32 log message should be shown:
-```
-ASTARTE_EXAMPLE_AGGREGATE: Astarte device connected, session_present: 0
-```
-You can see how the connection has been achieved also from the Astarte Dashboard by opening the
-newly created device and observing the status that should have changed to *Connected*.
+You flashed your device with the example firmware and connection between the device and the Astarte
+instance has been achieved.
```

Comment:
(Probabily unnecessary) Rewording suggestion: _established_ instead of _achieved_.